maintenance/maintenance_test.go

```diff
@@ -24,6 +24,7 @@ func stacks() []string {
 		"osx-xcode-11.0.x",
 		"osx-xcode-11.1.x",
 		"osx-xcode-11.2.x",
+		"osx-xcode-11.3.x",
```

Comment:
pls mark this update somewhere, like on the card if we have for it as resolved

Reply:
I mean, no. This wold silent our notification to update the stack list in the scanner.

Reply:
please remove this modification from this PR.